Imageview/app/src/main/java/com/mycompany/imageview/MySQLiteHelper.java

```diff
@@ -0,0 +1,180 @@
+package com.mycompany.imageview;
+
+/**
+ * Created by root on 9/28/15.
+ */
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import java.io.File;
+import java.util.LinkedList;
+import java.util.List;
+
+public class MySQLiteHelper extends SQLiteOpenHelper {
+
+    // Database Version
+    private static final int DATABASE_VERSION = 2;
+    // Database Name
+    private static final String DATABASE_NAME = "Image_Library";
+
+    // Contacts table name
+    private static final String TABLE_CONTACTS = "Images";
+    // Image Table Columns names
+    private static final String KEY_ID = "id";
+    private static final String KEY_keyword = "keyword";
+    private static final String KEY_link = "link";
+
+    private static final String[] COLUMNS = {KEY_ID,KEY_keyword,KEY_link};
```

Comment:
Any reason for not using a database contract and instead placing everything into the same helper?